packages/string-templates/test/javascript.spec.js

```diff
@@ -122,13 +122,6 @@ describe("Test the JavaScript helper", () => {
     const output = processJS(`return process`)
     expect(output).toBe("Error while executing JS")
   })
-
-  it("should prevent sandbox escape", () => {
```

Comment:
This test depends on the implementation. vm2 would crash, but VM does not. Probably it's better to remove it for now

Reply:
I would probably keep this test but `x.it` it out, we definitely want to confirm stuff like this on the backend, perhaps we should just move this test to the server?